app/models/upload.rb

```diff
@@ -51,6 +55,14 @@ def access_control_post
 
   scope :by_users, -> { where("uploads.id > ?", SEEDED_ID_THRESHOLD) }
 
+  def self.verification_statuses
+    @verification_statuses ||= Enum.new(
+      not_checked: 1,
+      verified: 2,
+      not_verified: 3
```

Comment:
Just curious, what is the difference between `not_checked` and `not_verified`? Those two keys seem to have the similar meaning to me :grinning:

Reply:
....foiled again! I think you are right. Maybe `unchecked`? The first value means we have not yet gone and checked the S3 inventory for that upload. Not verified means the upload etag does not match after we have checked.

Reply:
Maybe `unverified`, `verified` and `invalid`? Does that seem to be clearer while still holding the original meaning?

Reply:
unverified reads as the opposite of verified for me. I think i will do `unchecked`, `verified`, and `invalid_etag` to be more specific. (maybe in future then we can have more than one invalid state)

Reply:
That is better :+1: Anyway, the keys are pretty flexible since it only affects the ruby side code.